packages/components/src/navigator/navigator-button/component.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+// eslint-disable-next-line no-restricted-imports
+import type { Ref } from 'react';
```

Comment:
One more here 🧹 

```suggestion
import type { ForwardedRef } from 'react';
```

Reply:
Woops! Done in https://github.com/WordPress/gutenberg/pull/38634/commits/47f4b1fd6321c76a19096e917242a9e8eb523d9a